src/client/pydaos/raw/daos_cref.py

```diff
@@ -99,6 +99,30 @@ class PoolInfo(ctypes.Structure):
                 ("pi_space", PoolSpace),
                 ("pi_rebuild_st", RebuildStatus)]
 
+
+class DaosPropertyEntry(ctypes.Structure):
+    _fields_ = [("dpe_type", ctypes.c_uint32),
+                ("dpe_reserv", ctypes.c_uint32),
+                ("dpe_val", ctypes.c_uint64)]
+
+
+class DaosProperty(ctypes.Structure):
+    _fields_ = [("dpp_nr", ctypes.c_uint32),
+                ("dpp_reserv", ctypes.c_uint32),
+                ("dpp_entries", ctypes.POINTER(DaosPropertyEntry))]
+
+    def __init__(self, num_structs):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class 'Structure' is not called